enterprise/frontend/src/embedding-sdk/components/private/SdkGlobalStylesWrapper.tsx

```diff
@@ -18,13 +19,16 @@ export function SdkGlobalStylesWrapper({
   ...divProps
 }: SdkContentWrapperProps & HTMLAttributes<HTMLDivElement>) {
   const fontFiles = useSelector(getFontFiles);
-
+  const theme = useMantineTheme();
   return (
-    <SdkGlobalStylesInner
-      baseUrl={baseUrl}
-      fontFiles={fontFiles}
-      {...divProps}
-    />
+    <>
+      <SdkGlobalStylesInner
+        baseUrl={baseUrl}
+        fontFiles={fontFiles}
+        {...divProps}
+      />
+      <Global styles={getMetabaseCssVariables(theme)} />
```

Comment:
I tried using `:root` with `css` but it doesn't seem to work since they're still part of a div. Maybe that's why, so I need to use `Global` here